src/store/juju/types.ts

```diff
@@ -53,7 +54,15 @@ export type ModelsList = {
   [uuid: string]: ModelListInfo;
 };
 
+export type CrossModelQueryState = {
+  results: CrossModelQueryResponse["results"] | null;
+  errors: CrossModelQueryResponse["errors"] | null;
```

Comment:
These would need to refer to the object type using the name that you set in jimm-facade and the errors type would also need to support a string.

```suggestion
  results: CrossModelQueryFullResponse["results"] | null;
  errors: CrossModelQueryFullResponse["errors"] | string | null;
```